web-services/query-websocket/src/main/java/datawave/webservice/websocket/QueryWebsocket.java

```diff
@@ -45,7 +43,7 @@
  * <strong>NOTE: </strong> This uses vendor-specific security extensions to work around a websocket specification hole. See <a
  * href="https://java.net/jira/browse/WEBSOCKET_SPEC-238">WEBSOCKET_SPEC-238</a> for more details.
  */
-@ServerEndpoint(value = "/{logic-name}", encoders = {QueryResponseMessageJsonEncoder.class}, decoders = {JsonQueryMessageDecoder.class},
+@ServerEndpoint(value = "/{logic-name}", encoders = {QueryResponseMessageJsonEncoder.class}, // decoders = {JsonQueryMessageDecoder.class},
```

Comment:
Did you mean to remove the decoders

Reply:
No, this was commented out from when more code was commented out, but since that code was uncommented/re-worked to compile per other MR comments, this no longer needs to be commented out.  I've put it back.